src/raymath.h

```diff
@@ -401,6 +408,13 @@ RMDEF float Vector3Length(const Vector3 v)
     return result;
 }
 
+// Calculate vector square length
+RMDEF float Vector3SqrLength(const Vector3 v)
```

Comment:
Please, could you review naming to `Vector3LengthSqr()` to better align with `Vector3Length()`?